pytorch_toolkit/human_pose_estimation/README.md

```diff
@@ -1,59 +1,80 @@
 
-<p align="center">
-  <img src="data/preview.jpg" />
-</p>
+![](data/preview.jpg)
 
 ## Table of Contents
 
 * [Requirements](#requirements)
 * [Prerequisites](#prerequisites)
 * [Training](#training)
 * [Validation](#validation)
-* [Pre-trained model](#pre-trained-model)
-* [Output format](#output-format)
-* [C++ demo](#cpp-demo)
-* [Python demo](#python-demo)
-* [Fine-tuning](#fine-tuning)
+* [Pretrained Model](#pre-trained-model)
+* [Output Format](#output-format)
+* [C++ Demo](#cpp-demo)
+* [Python Demo](#python-demo)
+* [Fine-Tuning](#fine-tuning)
 * [Citation](#citation)
 
 ## Requirements
 
-* Ubuntu 16.04
-* Python 3.6
-* PyTorch 0.4.1 (should also work with 1.0, but not tested)
+* Ubuntu\* 16.04
+* Python\* 3.6
+* PyTorch\* 0.4.1 (PyTorch\* 1.0 not validated)
 
 ## Prerequisites
 
-1. Download COCO 2017 dataset: [http://cocodataset.org/#download](http://cocodataset.org/#download) (train, val, annotations) and unpack it to `<COCO_HOME>` folder.
-2. Create virtual environment `bash init_venv.sh`
+1. Download the COCO 2017 dataset: [http://cocodataset.org/#download](http://cocodataset.org/#download) (train, val, annotations) and unpack it to the `<COCO_HOME>` folder.
+2. Create virtual environment:
+    ```bash
+      bash init_venv.sh
+    ```
 
 ## Training
 
-Training consists of 3 steps (given AP values for full validation dataset):
-* Training from MobileNet weights. Expected AP after this step is ~38%.
-* Training from weights, obtained from previous step. Expected AP after this step is ~39%.
-* Training from weights, obtained from previous step and increased number of refinement stages to 3 in network. Expected AP after this step is ~40% (for the network with 1 refinement stage, two next are discarded).
+Training consists of three steps (given AP values for full validation dataset):
+1. Training from MobileNet weights. Expected AP after this step is ~38%.
+2. Training from weights obtained from the previous step. Expected AP after this step is ~39%.
+3. Training from weights obtained from the previous step and increased number of refinement stages to 3 in the network. Expected AP after this step is ~40% (for a network with one refinement stage, two next ones are discarded).
 
-1. Download pre-trained MobileNet v1 weights `mobilenet_sgd_68.848.pth.tar` from: [https://github.com/marvis/pytorch-mobilenet](https://github.com/marvis/pytorch-mobilenet) (sgd option).
 
-2. Convert train annotations in internal format. Run `python scripts/prepare_train_labels.py --labels <COCO_HOME>/annotations/person_keypoints_train2017.json`. It will produce `prepared_train_annotation.pkl` with converted in internal format annotations.
+### Training Flow
+1. Download the pretrained MobileNet v1 weights `mobilenet_sgd_68.848.pth.tar` from the [Pytorch*-MobileNet repository](https://github.com/marvis/pytorch-mobilenet) (choose the sgd option).
```

Comment:
`pytorch-mobilenet` is the name of repository (not a ml library), so suggest to leave the name as is.